Airports/EGPH/SMAA.txt

```diff
@@ -90,12 +90,12 @@ EGPH Edinburgh SMAA         S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00
                             N055.53.21.000 W002.56.08.000 N055.58.43.000 W003.17.08.000
                             N055.58.43.000 W003.17.08.000 N055.56.52.000 W003.49.58.000
 ;FAVA06
-                            N055.56.58.390 W003.26.38.759 N055.54.51.030 W003.24.22.565
-                            N055.54.51.030 W003.24.22.565 N055.49.39.881 W003.35.26.133
-                            N055.49.39.881 W003.35.26.133 N055.53.54.600 W003.39.58.522
-                            N055.53.54.600 W003.39.58.522 N055.56.58.390 W003.26.38.759
+							N055.56.44.103 W003.27.26.461 N055.54.35.730 W003.25.07.962
+							N055.54.35.730 W003.25.07.962 N055.49.23.249 W003.36.10.576
+							N055.49.23.249 W003.36.10.576 N055.53.39.994 W003.40.47.081
+							N055.53.39.994 W003.40.47.081 N055.56.44.103 W003.27.26.461
```

Comment:
```suggestion
                            N055.56.44.103 W003.27.26.461 N055.54.35.730 W003.25.07.962
                            N055.54.35.730 W003.25.07.962 N055.49.23.249 W003.36.10.576
                            N055.49.23.249 W003.36.10.576 N055.53.39.994 W003.40.47.081
                            N055.53.39.994 W003.40.47.081 N055.56.44.103 W003.27.26.461
```

Reply:
ok fine but the rest of the file uses tabs.....

Reply:
fixed